paper/writeup.tex

```diff
@@ -682,32 +687,45 @@ \subsection{Dissimilarity between ARGs}
     \alpha(n_1) = \argmax_{n_2 \in N_2} m(n_1, n_2) ,
 \end{align*}
 so that
-\begin{align*}
-    \similarity(\T_1, \T_2)
+\begin{align}
+    \match(\T_1, \T_2)
     =
     \sum_{n \in N_1} m(n, \alpha(n)) .
-\end{align*}
+\end{align}
 If the best-matching node is not unique, we define $\alpha(n_1)$ to be the node in $T_2$
 out of those maximizing $m(n_1, n_2)$ that minimizes $|t^{(1)}_{n_1} - t^{(2)}_{n_2}|$
 (and if \emph{this} is not unique, we pick an arbitrary one) --
-however, this potential ambiguity does not affect the definition of $\similarity(\T_1, \T_2)$.
-Let $\|\T_1\|=\sum_{n\in N_1}s(\T_1,n)$ be the total sum of nodes spans for nodes in $\T_1$,
-and let $s(\T,n)$ denote
-the total span that node $n$ is present in the local trees,
+however, this potential ambiguity does not affect the definition of $\match(\T_1, \T_2)$.
+Let $s(\T,n)$ denote the total span that node $n$ is present in the local trees,
 \begin{align*}
-    s(\T, n) = \sum_{k=1}^N (a_k - a_{k-1}) \ind_{n \in T_k} \quad ,
+	s(\T, n) = \sum_{k=1}^N (a_k - a_{k-1}) \ind_{n \in T_k} \quad ,
 \end{align*}
-where $\ind_{n \in T_k}$ is an indicator (i.e., it is 1 if $n \in T_k$ and 0 otherwise).
-We then define the \emph{dissimilarity} of $\T_1$ from $\T_2$ by
+where $\ind_{n \in T_k}$ is an indicator (i.e., it is 1 if $n \in T_k$ and 0 otherwise),
+and let $\|\T_1\|=\sum_{n\in N_1}s(\T_1,n)$ be the total sum of nodes spans for nodes in $\T_1$.
+We then define the \emph{non-matched span} of $\T_1$ in $\T_2$ by
 \begin{align*}
-    \dis(\T_1, \T_2)
-    =
+	\nmatch(\T_1,\T_2)
+	=
     \sum_{n \in N_1} (s(\T_1, n) - m(n, \alpha(n))) 
     = 
-    \Vert \T_1\Vert -\similarity(\T_1,\T_2),
+    \Vert \T_1\Vert -\match(\T_1,\T_2),
 \end{align*}
 which is the total span for all nodes in $\T_1$
-over which their descendant samples do \emph{not} match those of their best match in $\T_2$.
+over which their descendant samples do \emph{not} match
+ those of their best match in $\T_2$.
+Contrarily, given a matching $\alpha:\T_1\to\T_2$, 
+we want to quantify how much of $\T_2$ is represented in $\T_1$.
+To do this, we define the \emph{inverse matched span} of $\T_1$ in $\T_2$ as
+\begin{align}
+	\invmatch(\T_1,\T_2) = \sum_{n_2\in N_2} \max_{n_1\in \alpha^{-1}(n_2)} m(n_1,n_2)
+\end{align}
+where $\alpha^{-1}(n_2)$ is the set of all nodes $n_1\in \T_1$ 
+whose best match is $n_2$.
+Note that the inverse matched span is not equivalent
+ to the matched span of $\T_2$ in $\T_1$ 
+ as the inverse matched span maximizes over spans
+  which are still dependent on $\beta$, 
+  and, as previously stated, $\beta$ is not guaranteed to be bijective.
```

Comment:
```suggestion
This differs from the matched span of $\T_2$ in $\T_1$
because there may be more than one node in $\T_1$
that is mapped to the same node in $\T_2$ --
so, if nodes $n_1$ and $n_1'$ are both matched by $\alpha$ to the same node $n_2$,
then both count towards $\match(\T_1,\T_2)$,
but only the better match counts towards $\invmatch(\T_1,\T_2)$.
```
we're going to Genetics, after all

Reply:
I figured "since it's not bijective" might need a bit more explanation for Genetics (and honestly to help out my future self).